torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -316,6 +363,26 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
   if (!py::hasattr(py_module_, field.c_str())) {
     throw ErrorReport(loc) << "module has no attribute '" << field << "'";
   }
+
+  auto is_mod_dict = py::isinstance(
+      py_module_, py::module::import("torch.jit").attr("_ConstModuleDict"));
+  if (is_mod_dict) {
+    if (field == "items" || field == "keys" || field == "values") {
+      bool get_keys = false;
+      bool get_values = false;
+      if (field == "items") {
+        get_keys = true;
+        get_values = true;
+      } else if (field == "values") {
+        get_values = true;
+      } else {
+        get_keys = true;
+      }
+      return std::make_shared<ConstantTupleMethod>(
```

Comment:
Why is this a `ConstantTupleMethod` if `ModuleDict`s can't be called?

Reply:
What do you mean? It's not being called, a function on it is being called.